driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -185,7 +185,7 @@ fn write_input(input_file: &str, input: &serde_json::Value) -> std::io::Result<(
 
 fn run_solver(input_file: &str) -> Result<(), PriceFindingError> {
     let output = Command::new("python")
-        .arg("./batchauctions/scripts/e2e/_run.py")
+        .arg(&["-m", "batchauctions.scripts.e2e._run"])
```

Comment:
either this should be `args` or two .arg calls (one with -m and the other one with the module). Cf. https://doc.rust-lang.org/std/process/struct.Command.html